README.md

```diff
@@ -49,4 +49,4 @@ echo $cssToInlineStyles->convert(
 * [Each site based on Fork CMS](http://www.fork-cms.com)
 * [Print en Bind](http://www.printenbind.nl)
 * [Tiki Wiki CMS Groupware](http://sourceforge.net/p/tikiwiki/code/49505) (starting in Tiki 13)
-* [Laravel Framework] (https://github.com/laravel/framework/blob/086c02a544af2870c52976408cded8cf59d7cf40/src/Illuminate/Mail/Markdown.php#L52)
+* [Laravel Framework](https://github.com/laravel/framework/blob/086c02a544af2870c52976408cded8cf59d7cf40/src/Illuminate/Mail/Markdown.php#L52)
```

Comment:
Maybe update the link to `https://github.com/laravel/framework/blob/v6.18.24/src/Illuminate/Mail/Markdown.php#L55`. Takes people to a recognisable, tagged, LTS version of the code.

Reply:
Done.